### PR TITLE
[WIP] Update smart proxy packages for SCL

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec [hunk 1]
```diff
@@ -75,6 +76,9 @@ cp -a .%{gem_dir}/* \
 %exclude %{gem_instdir}/Gemfile*
 
 %changelog
+* Fri Sep 06 2019 Eric D. Helms <ericdhelms@gmail.com> - 0.10.0-4
+- Updates to build for SCL
```
Comment (ekohl) [inline packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec]: I don't see any real updates in this file. You could clean the EL6 compatibility code if you want to make changes :)

File: packages/foreman/rubygem-rb-inotify/rubygem-rb-inotify.spec [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # Generated from rb-inotify-0.9.7.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name rb-inotify
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
```
Comment (ekohl) [inline packages/foreman/rubygem-rb-inotify/rubygem-rb-inotify.spec]: Perhaps you can clean the EL6 compatibility macros from this file while you're at it? That also means the `%doc` can become `%license`.

**Comment by ehelms** [comment]
I'll update the spec file for FFI, but when we finally SCL we are gonna have to move FFI from plugins to main repository and build it for SCL.

**Comment by ehelms** [comment]
This is now just a place to store all of the changes necessary to SCLize the entire foreman proxy package set. I will be pulling off commits to get them merged individually.

**Pull request closed without merging**